src/test/java/org/apache/sysds/test/AutomatedTestBase.java

```diff
@@ -1263,16 +1267,98 @@ protected void runRScript(boolean newWay) {
 		}
 	}
 
-	private static String buildOutputStringR(String standardOut, String standardError){
+	/**
+	 * Runs an Python script
+	 */
+	protected void runPythonScript() {
```

Comment:
this method looks a bit funky, it could probably be cleaned up a bit.

Reply:
I have written the `runPythonScript()` function by following the structure of the existing `runRScript()` function to run Python scripts. However, I would like to ask for clarification on how you would like me to proceed with further modifications.

Would you prefer that I refactor the code to combine the `runRScript()` and `runPythonScript()` functions into a single function (e.g., `runBaselineScript()`), and also consolidate class variables such as `rCmd`?